modules/sd-nodes-cma-autoscaling-custom-install.adoc

```diff
@@ -0,0 +1,129 @@
+is selected for *Installation Mode*.
+. For *Installed Namespace*, click *Select a namespace*.
+
+. Click *Select Project*: 
++
+* If the `keda` namespace exists, select *keda* from the list.
+* If the `keda` namespace does not exist: 
++
+.. Select *Create Project* to open the the *Create Project* window. 
+.. In the *Name* field, enter `keda`. 
+.. In the *Display Name* field, enter a descriptive name, such as `keda`.  
+.. Optional: In the *Display Name* field, add a description for the namespace.
+.. Click *Create*.    
+endif::openshift-rosa,openshift-dedicated[]
+
+. Click *Install*.
+
+. Verify the installation by listing the Custom Metrics Autoscaler Operator components:
+
+.. Navigate to *Workloads* -> *Pods*.
+
+.. Select the `keda` project from the drop-down menu and verify that the `custom-metrics-autoscaler-operator-*` pod is running.
+
+.. Navigate to *Workloads* -> *Deployments* to verify that the `custom-metrics-autoscaler-operator` deployment is running.
+
+. Optional: Verify the installation in the OpenShift CLI using the following commands:
++
+[source,terminal]
+----
+$ oc get all -n keda
+----
++
+The output appears similar to the following:
++
+.Example output
+[source,terminal]
+----
+NAME                                                      READY   STATUS    RESTARTS   AGE
+pod/custom-metrics-autoscaler-operator-5fd8d9ffd8-xt4xp   1/1     Running   0          18m
+
+NAME                                                 READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/custom-metrics-autoscaler-operator   1/1     1            1           18m
+
+NAME                                                            DESIRED   CURRENT   READY   AGE
+replicaset.apps/custom-metrics-autoscaler-operator-5fd8d9ffd8   1         1         1       18m
+----
+
+. Install the `KedaController` custom resource, which creates the required CRDs:
+
+.. In the {product-title} web console, click *Operators* -> *Installed Operators*.
+
+.. Click *Custom Metrics Autoscaler*.
+
+.. On the *Operator Details* page, click the *KedaController* tab.
+
+.. On the *KedaController* tab, click *Create KedaController* and edit the file.
++
+[source,yaml]
+----
+kind: KedaController
+apiVersion: keda.sh/v1alpha1
+metadata:
+  name: keda
+  namespace: keda
+spec:
+  watchNamespace: '' <1>
+  operator:
+    logLevel: info <2>
+    logEncoder: console <3>
+  metricsServer:
+    logLevel: '0' <4>
+    auditConfig: <5>
+      logFormat: "json"
+      logOutputVolumeClaim: "persistentVolumeClaimName"
+      policy:
+        rules:
+        - level: Metadata
+        omitStages: "RequestReceived"
+        omitManagedFields: false
+      lifetime:
+        maxAge: "2"
+        maxBackup: "1"
+        maxSize: "50"
+  serviceAccount: {}
+----
+<1> Specifies the namespaces that the custom autoscaler should watch. Enter names in a comma-separated list. Omit or set empty to watch all namespaces. The default is empty.
```

Comment:
This reminds me... The `watchNamespace` field only supports one namespace or all namespaces (blank). It doesn't support a comma-separated list. Sorry about that. If you would prefer, I can file a separate bug to track that since it needs to be fixed in the ROSA and the vanilla section of the docs and isn't the topic of this PR.

Reply:
@joelsmith I updated it here, might as well. 

> Specifies a single namespace where the custom autoscaler is to scale applications. Omit or set empty to scale applications in all namespaces. The default is empty.